skfp/preprocessing/filters/rule_of_2.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+
+from rdkit.Chem import Mol
+from rdkit.Chem.Crippen import MolLogP
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumHBA, CalcNumHBD
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class RuleOf2(BaseFilter):
```

Comment:
I think we should avoid numbers in class names, so this would be `RuleOfTwo`

Reply:
Done